galasa-managers-parent/galasa-managers-core-parent/dev.galasa.textscan.manager/src/main/java/dev/galasa/textscan/LogScanner.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed Materials - Property of IBM
+ * 
+ * (c) Copyright IBM Corp. 2020.
+ */
+package dev.galasa.textscan;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import dev.galasa.framework.spi.ValidAnnotatedFields;
+
+/**
+ * <p>
+ * Fill this field with a Log Scanner object.
+ * </p>
+ *
+ * <p>
+ * Will only populate public {@link ILogScanner} fields.
+ * </p>
+ *
+ * @author Michael Baylis
+ * @see {@link ILogScanner}
+ *
+ */
+@Retention(RUNTIME)
+@Target(FIELD)
+@TextScanManagerField
+@ValidAnnotatedFields({ ILogScanner.class })
```

Comment:
How can we do this when we might not have access to all the things we want to scan when the test is written

Reply:
because you call setScanned or updateText

Reply:
Just seems strange to provision an object like a LogScanner that doesn't have an analog that it represents, why not just get an instance of the LogScanner from File or Batch Manager

Reply:
because you may be scanning a text file requiring checkpointing that is not provided by a manager.   It could be a text file that the test itself is monitoring,  so need to provide a way in